pkgs/by-name/di/difftastic/package.nix

```diff
@@ -5,50 +5,37 @@
   fetchFromGitHub,
   testers,
   difftastic,
+  nix-update-script,
 }:
 
-let
-  mimallocPatch = fetchpatch {
-    # fixes compilation error on x86_64-darwin
-    # remove after update to libmimalloc-sys >= 0.1.29
-    # (fixed in mimalloc >= 1.7.6 which is included with libmimalloc-sys >= 0.1.29)
-    url = "https://github.com/microsoft/mimalloc/commit/40e0507a5959ee218f308d33aec212c3ebeef3bb.patch";
-    hash = "sha256-DK0LqsVXXiEVQSQCxZ5jyZMg0UJJx9a/WxzCroYSHZc=";
-  };
-in
 rustPlatform.buildRustPackage rec {
   pname = "difftastic";
-  version = "0.62.0";
+  version = "0.63.0";
 
   src = fetchFromGitHub {
     owner = "wilfred";
     repo = pname;
-    rev = version;
-    hash = "sha256-AAnlopJTb+tjkACISK9OC0k59BDt+8michzt37P1lL8=";
+    tag = version;
+    hash = "sha256-BxWCSkSeDyiiGBY2u0ahPrIhYq2lbujoPPtZGq/OkI0=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
```

Comment:
Please also delete the `Cargo.lock` file

Reply:
Will do!